script/deploy-alchemy-subgraph.js

```diff
@@ -0,0 +1,17 @@
+// script for deploying subgraph to Alchemy
+// Usage: node script/deploy-alchemy-subgraph.js <network>
+require('dotenv').config();
+const shell = require('shelljs');
+
+const network = process.argv[2];
+const deployKey = process.env[`${network.toUpperCase()}_DEPLOY_KEY`];
```

Comment:
Could we update the .env.example with the networks we currently support on Umbra?

Reply:
Currently the .env.example only contains and indication that the  X_DEPLOY_KEY is needed for Base, its subgraph is deployed to Alchemy, and the subgraphs for all other chains are on goldsky and the authentication for deployment there (as per the README file) is to do step 3 of `Authenticate with `yarn graph auth https://api.thegraph.com/deploy/ <access-token>`.

As we move other chains to Alchemy, we can add the indication of needed deploy key for them to .env.example

Reply:
Hi, since we've decided to move to Alchemy, let's add the other chains to `.env.example`.

We should also update the `package.json` scripts.

Reply:
Done: [29a17af](https://github.com/ScopeLift/umbra-subgraph/pull/7/commits/29a17af940226cbdb3c58c43071dabc6d12d2aac)